upstream fix for TL2016 luatex and graphicx git commit 2f209f9ed847
From: John Bowman <bowman@ualberta.ca>
Date: Sun, 28 Aug 2016 23:36:02 -0600
Subject: [PATCH] Implement general workarounds for recently introduced
 graphicx and lualatex backwards incompatibilities.
---
 base/plain.asy |    7 +++++++
 1 file changed, 7 insertions(+)

--- asymptote.orig/base/plain.asy
+++ asymptote/base/plain.asy
@@ -280,3 +280,10 @@
 }
 
 cputime();
+
+texpreamble("\newdimen\paperwidth");
+texpreamble("\newdimen\paperheight");
+texpreamble("\ifx\pdfpagewidth\undefined\let\pdfpagewidth\paperwidth\fi");
+texpreamble("\ifx\pdfpageheight\undefined\let\pdfpageheight\paperheight\fi");
+if(settings.tex == "luatex") texpreamble("\input luatex85.sty");
+
